# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unixodbc/new_bison.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

YY_FLUSH_BUFFER is a macro for yy_flush_buffer which is not available on this scope.
I have no idea yet how safe is this but i want to have it built first

Alejandro Mery (2005-03-18)

--- ./sqp/lex.l.orig	2001-10-17 13:40:32.000000000 -0300
+++ ./sqp/lex.l	2005-03-18 11:28:24.155640480 -0400
@@ -236,8 +236,6 @@
 void yyerror( char *s )
 {
 	sprintf( g_szError, "%s processing '%s' on line %d", s, yytext, g_nLineNo );
-	
-	YY_FLUSH_BUFFER;
 }
 
 /**********************************************************************
